### ISSUE TITLE
[featureRequest] posibility to dualize partial sections of mesh

### ISSUE THREAD

**Issue by otaolafranc** [MoiseRousseau/SALOME-Voronoi#8]
It would be awesome to be able to introduce a hybrid mesh that has tetras inside and dualize this section to poly without modifying the rest of the elements of the mesh (and simply keeping them as they were).
one dummy example of a case like this:
[example.tar.gz](https://github.com/MoiseRousseau/SALOME-Voronoi/files/14374186/example.tar.gz)

thanks in advance,
regards,

**Comment by MoiseRousseau** [comment]
The problem is how the dual / Voronoi mesh will share the same faces and vertices at the interface of the two sections ? This is a complicated math problem and the Geogram library does not allow that. So this is not possible with the current plugin unfortunately.

**Comment by otaolafranc** [comment]
thanks for the insight @MoiseRousseau , I thought of the posibility of adding the extra nodes/edges to the faces of the boundaries with the tetra cells. something like it is done in snappyHexMesh 4.4.2 (https://www.openfoam.com/documentation/user-guide/4-mesh-generation-and-conversion/4.4-mesh-generation-with-the-snappyhexmesh-utility) where you add extra vertex to hexahedral mesh to get a conformal connection between a hex cell and 4 hex cells on one of its face. (the 'big' hex cell it would be actually a poly cell which 'looks' like an hex one)

**Comment by MoiseRousseau** [comment]
Yes true. But my intent was not to create a mesher with SALOME-Voronoi but rather an interface with GeoGram. Also, I don't have more time to put on that. If you want to enhance my work, please fork the repo and create your own mesher! :)
